CHANGELOG.md

```diff
@@ -3,11 +3,12 @@
 Version 5.0.0                      2019-03-27
 ---------------------------------------------
 - New Webpack Build System
-- Two revamped and improved Grid Systems
+- Two revamped and improved Grid Systems with 6 breakpoints
 - Flexbox based, class generated grid system
 - CSS Grid based mixin supported grid system
```

Comment:
Should lines 7 & 8 be indented relative to line 6?

Reply:
Sure, I think that makes sense.